module/module.py

```diff
@@ -341,6 +344,18 @@ def __init__(self, modconf):
         logger.info("[WebUI] parameter timezone: %s", self.timezone)
 
         # Visual alerting thresholds
+        # --------------------------
+        # All the hosts and services that are in a HARD non OK/UP state are considered as problems if their
+        # business_impact is greater than or equal this value
+        self.min_business_impact = int(getattr(modconf, 'min_business_impact', '0'))
+        # badge_business_impact is used to filter the alerting badges in the header bar (default is 2)
+        self.badge_business_impact = int(getattr(modconf, 'badge_business_impact', '2'))
+        logger.info("[WebUI] minimum business impacts, all UI: %s, badges: %s",
+                    self.min_business_impact, self.badge_business_impact)
+
+        # Inner computation rules for the problems
+        self.inner_problems_count = int(getattr(modconf, 'inner_problems_count', '1'))
```

Comment:
I keep this option configurable because it may be interesting to have a view on the real computation done by the framewrok. 
I will rename as `disable_inner_problems_computation`